TestModels/Resource/runtimes/rust/src/types/i_simple_resource.rs

```diff
@@ -0,0 +1,56 @@
+// Code generated by software.amazon.smithy.rust.codegen.smithy-rs. DO NOT EDIT.
+
+pub trait ISimpleResource {
```

Comment:
The "I" prefix doesn't seem very idiomatic for Rust, I think we should drop it.

Reply:
Done